pkg/cmd/server/origin/master.go

```diff
@@ -559,9 +561,11 @@ func (c *MasterConfig) checkProjectRequestTemplate() {
 		return
 	}
 
-	tokens := strings.Split(c.Options.ProjectRequestConfig.ProjectRequestTemplate, "/")
-	namespace := tokens[0]
-	templateName := tokens[1]
+	namespace, templateName, err := configapi.ParseNamespaceAndName(c.Options.ProjectRequestConfig.ProjectRequestTemplate)
+	if err != nil {
+		glog.Errorf("Error creating default project request template %v.  Unprivileged project requests will fail until a template is available.  Caused by: %v", c.Options.ProjectRequestConfig.ProjectRequestTemplate, err)
+		return
+	}
 
 	if _, err := c.PrivilegedLoopbackOpenShiftClient.Templates(namespace).Get(templateName); !kapierror.IsNotFound(err) {
```

Comment:
since `ParseNamespaceAndName` can return `"","",nil`, seems like checking len(namespace) and len(templateName) should take the place of the len(c.Options.ProjectRequestConfig.ProjectRequestTemplate) check above